### ISSUE TITLE
Layering problems

### ISSUE THREAD

**Issue by Kammerjunk** [d3athrow/vgstation13#10635]
#### Description of issue

![layers](https://cloud.githubusercontent.com/assets/5942183/16173935/ab741c64-35b0-11e6-929a-772ff4e4b363.png)
Some things are layered in ways they shouldn't be. Respectively to the picture above,

> The Blocker objects in the Thunderdome overlay the windows
> The janitor overlays the janicart
> Outer clothing storage sprites overlay items stored
> Blood overlays some things

Tested on 510.1347
#### Difference between expected and actual behavior

Mostly aesthetic, I guess, but not being able to pick up whatever you've put in your labcoat or hazard suit or coat could be a bit of a mess.
#### Steps to reproduce

Obtain labcoat/hazard vest
Wear it, open it
Put a(n) (tiny?) item into one of the storage slots
Attempt to retrieve it again
#### Length of time in which bug has been known to occur

Decently recent. Maybe for, oh, 21 hours? #10499 
#### Server revision

df0258454d3cabecc8ebfca9d7095cc5c1a4995f
#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

**Comment by clusterfack** [comment]
@drcelt
![](http://www.thedailysheeple.com/wp-content/uploads/2014/09/9-11-survivors-pasquale-buzzelli-who-survived-after-39surfing39-on-2.jpg)

**Comment by Shadowmech88** [comment]
I noticed yesterday someone was holding a fireaxe, and no matter what direction they faced, the fireaxe seemed to be layered below them.

This made it _really_ easy for them to conceal the fireaxe.

**Comment by Intigracy** [comment]
Thanks Celt.

**Comment by ghost** [comment]
I guarantee you that 99% of the bugs because of planes was because of old coders.

**Comment by ghost** [comment]
I'll fix them very quickly, should be simple.

**Comment by 9600bauds** [comment]
> should be simple.

famous last words

**Comment by Intigracy** [comment]
> because of old coders

Uh if you were reworking how things worked it's your fault you didn't rework the old code not the old coders wtf.

**Comment by ghost** [comment]
@Intigracy Unless you expect me to test literally everything that can occur in the game, *shrug, I'm not sure what you mean. There are standardised ways of doing things. I looked into both the standardised way of dealing with layers and the non standardised ways of doing things that I came across, and tested relatively extensively, see #10499 . If some coder decides to go out of his way to rewrite the entire wheel, I can hardly be blamed for when it gets a puncture under the weight of a new system when I didn't even know it existed, and could not have known it existed without being previously familiar with it.

in short: magic numbers are awful, and I hate them

**Comment by Gringonius** [comment]
Not sure if related but soot from fires block clicking the actual tiles below it unless you alt-click while near, making repiping a room with those a huge pain in the ass if you don't have soap.
